CHANGES_NEXT_RELEASE

```diff
@@ -22,3 +22,4 @@
 - Fix: no longer adding subscriptions from all tenants in the cache if broker isn't multitenant (Issue #1555)
 - Fix: resetting temporal counters at synching the subscription cache (Issue #1562)
 - Fix: using the text "too many sbuscriptions" at cache statistics operations in the case of too many results
+- Fix: crash due to subscription ID not conforming to supposed syntax (Issue #1552)
```

Comment:
- Fix: crash due to subscription ID not conforming to supposed syntax in GET /v2/subscriptions/<id> operation (Issue #1552)

The "in GET /v2/subscriptions/<id> operation" could be relevant given that, as far as I understand, this problem doesn't happen in similar operations of the NGSIv1 family.